core/tests/test_trezor.protobuf.py

```diff
@@ -1,7 +1,7 @@
 from common import *
 
 from trezor import protobuf
-from trezor.messages import WebAuthnCredential, EosAsset, Failure, SignMessage
+from trezor.messages import WebAuthnCredential, Failure, SignMessage, DebugLinkMemoryRead
```

Comment:
I vaguely recall horrible things happening when we do that. but we might try at some point

Reply:
Could be that not all imports are at the top of the file (due to need of adding something to `path` to import stuff), so `isort`ing it might really fail. `Black` should be fine though